app/assets/stylesheets/application.css.scss

```diff
@@ -10,6 +10,7 @@
  * files in this directory. Styles in this file should be added after the last require_* statement.
  * It is generally better to create a new file per style scope.
  *
- *= require_self
- *= require main
  */
+ @import "bootstrap";
+ @import url("layouts/user_header.scss");
+ @import url("pages/home.scss");
```

Comment:
So given comments below by @arincon9 and @sanghun89 lets change this file back to .scss and we'll import each related scss file here as you've done. Minor point, but I don't think you need the `url()` piece though, `@import "layouts/user_header.scss";` should be sufficent.

Reply:
@uuunk I will take a look at these and do another commit, hopefully today, and let you know. 

I was able to look at the merge conflicts. I ran into some issues but @mintii walked me through most of them so hopefully we are in a better spot now. I have a few left so I will circle back to you guys if I need help but I will try to work through them on my own.